### ISSUE TITLE
Usability improvements

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/supertunnel#13]
These aren't WCAG violations, just things that real users generally prefer.

- [ ] Remove the `<header>` tags wrapping the headings in each section. Headers are most helpful when there is only one on the page, representing the global page header.
- [ ] Remove the `<figure>` tags wrapping the images, since all they do is add redundant noise to the screen reader user experience.
